README.md

```
@@ -1,3 +1,98 @@
 # java-janggi
 
-장기 미션 저장소
+> 장기의 보드판 그리고 각 기물과 움직임을 구현합니다. (현재 버전에서는 장군과 사는 구현하지 않습니다.)
+
+## 구현할 기능 목록
+
+- [x] 게임을 시작한다.
+    - [x] 환영 메시지와 게임 규칙을 출력한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)